app/Http/Controllers/Rest/CampaignsController.php

```diff
@@ -43,41 +45,31 @@ public function add(Request $request)
             }
         }
 
-        $reqObj = $request->input('site.targeting.exclude');
+        $reqObj = $request->input('campaign.targeting.exclude');
         if (null != $reqObj) {
             foreach (array_keys($reqObj) as $key) {
                 $value = $reqObj[$key];
                 $campaign->campaignExcludes()->create(['key' => $key, 'value' => $value]);
             }
         }
 
-        $response = self::json(compact('campaign'), 201);
-        $response->header('Location', route('app.campaigns.read', ['campaign' => $campaign]));
-
-        return $response;
+        return self::json(compact('campaign'), 201)
+            ->header('Location', route('app.campaigns.read', ['campaign' => $campaign]));
     }
 
     public function browse(Request $request)
     {
         // TODO check privileges
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1135)